bodhi/server/metadata.py

```diff
@@ -11,23 +11,21 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-
-__version__ = '2.0'
-
-import os
 import logging
+import os
 import shutil
 import tempfile
 
-from urlgrabber.grabber import urlgrab
 from kitchen.text.converters import to_bytes
-
+from urlgrabber.grabber import urlgrab
 import createrepo_c as cr
 
+from bodhi.server.buildsys import get_session
 from bodhi.server.config import config
 from bodhi.server.models import Build, UpdateStatus, UpdateRequest, UpdateSuggestion
-from bodhi.server.buildsys import get_session
 
+
+__version__ = '2.0'
 log = logging.getLogger(__name__)
```

Comment:
@pypingou ```flake8``` doesn't complain about this. I think pylint might, but I personally find that I disagree with pylint on many things (it goes beyond PEP-8). I only intend to enforce PEP-8 on Bodhi's code for now, with the exception that I allow a 100 character line length instead of 80 characters because c'mon, anyone on one of those old 80's monitors should upgrade to a 1080p screen, at minimum ☺ 100 chars is far more readable and encourages more meaningful variable names (like ```user``` instead of ```u```).

Additionally, capitalizing ```LOG``` would imply that it is a constant by PEP-8, and it is not a constant - it's a Logger instance ☺